Packs/Sysdig/Integrations/Sysdig/Sysdig.py

```diff
@@ -0,0 +1,337 @@
+             }
+    if csvOut == 'yes':
+        # Convert to csv
+        header = ''
+        row = ''
+        outputContents = ''
+
+        # Create csv header row
+        for item in results[0].keys():
+            header = header + item + ','
+        # Remove trailing comma
+        header = header[:-1] + '\n'
+
+        outputContents = header
+        for entry in results:
+            for value in entry.values():
+                row = row + str(value).replace(',', ';') + ','
+            row = row[:-1]
+            outputContents = outputContents + row + '\n'
+            row = ''
+        filename = 'container_vulnerabilities.csv'
+        file_content = outputContents
+        return_results(fileResult(filename, file_content))
+    else:
+        return entry
+
+
+def query_vulns_by_container():
+    imageId = demisto.args()['resultId']
+    limit = demisto.args()['limit']
+    results = []
+    url = URL + 'api/scanning/scanresults/v2/results/' + imageId + \
+        '/vulnPkgs?filter=&limit=' + str(limit) + '&offset=0&order=asc&sort=vulnSeverity'
+    token = 'Bearer ' + SECUREKEY
+    headers = {'Authorization': token}
+    response = requests.get(url=url, headers=headers).json()
+    for item in response['data']:
+        row = flatten_data(item)
+        results.append(row)
+    entry = {'Type': entryTypes['note'],
+             'Contents': results,
+             'ContentsFormat': formats['json'],
+             'HumanReadable': tableToMarkdown('Vulnerabilities Found', results, removeNull=True),
+             'ReadableContentsFormat': formats['markdown'],
+             'EntryContext': {'ContainerVulns': results}
+             }
+    return entry
+
+
+def test_module():
+    try:
+        sdclient = SdMonitorClient(token=MONITORKEY, sdc_url=URL)
+        ok, res = sdclient.get_alerts()
+        if ok is True:
+            demisto.results('ok')
+    except Exception as e:
+        LOG(e)
+        return_error(e.message)
+
+
+try:
+    if demisto.command() == 'test-module':
+        test_module()
+    elif demisto.command() == 'sysdig-get-alerts':
+        demisto.results(get_alerts())
+    elif demisto.command() == 'sysdig-get-metrics':
+        demisto.results(get_metrics())
+    elif demisto.command() == 'sysdig-get-users':
+        demisto.results(get_users())
+    elif demisto.command() == 'sysdig-get-events':
+        demisto.results(get_events())
+    elif demisto.command() == 'sysdig-list-policies':
+        demisto.results(list_policies())
+    elif demisto.command() == 'sysdig-list-hosts':
+        demisto.results(list_hosts())
+    elif demisto.command() == 'sysdig-list-vulnerabilities':
+        demisto.results(list_vulns())
+    elif demisto.command() == 'sysdig-list-vulns-by-container':
+        demisto.results(query_vulns_by_container())
+except Exception as e:
+    return_error('Error has occurred in the Sysdig Integration: {error}\n {message}'.format(error=type(e), message=e.message))
```

Comment:
I suggest you should work with a main() function to make your code more readable, you can use [this](https://github.com/demisto/content/blob/master/Packs/HelloWorld/Integrations/HelloWorld/HelloWorld.py) integration as a reference.